version.txt

```diff
@@ -21,6 +21,7 @@ car_eval.json,1.1.2
 catalogos_dados_brasil.json,1.0.0
 chytr_disease_distr.json,1.0.2
 community_abundance_misc.json,1.0.3
+coronaviruskoreadataset.json,1.0.0
```

Comment:
Let us change the name to `coronaviruskorea`
south_korea_coronavirus

Reply:
Fixed it, I agree coronaviruskoreadataset was a complicated name for the dataset